test/e2e/cluster_test.go

```diff
@@ -147,3 +148,40 @@ func TestClusterGet(t *testing.T) {
 
 	assert.Contains(t, output, `status: Successful`)
 }
+
+func TestClusterNameInRestAPI(t *testing.T) {
+	SkipIfAlreadyRun(t)
```

Comment:
No need of calling `SkipIfAlreadyRun(t)`, it is already called inside `EnsureCleanState`

Reply:
thank you! Removed